docs/user-guide/03-convenience-features.md

```diff
@@ -383,6 +383,70 @@ int caller()
 }
 ```
 
+---
+layout: user-guide
+permalink: /user-guide/convenience-features-if-let
+---
```

Comment:
What is these lines for?
I thought they appear only at the beginning of the documents once.

Reply:
This is incorrect and we should delete these lines.

Reply:
So you say this is wrong anchor?
I want to reference it in another chapter at here:
https://github.com/shader-slang/slang/pull/4628/files#diff-eb730bcc516fb44ca0442cb2f58e4197739f09f352ab72bb1bec7a88dfe1e1daR682

Reply:
I figure out the correct way to add anchor.